backend/public/index.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+$_SERVER['REQUEST_URI'] = preg_replace('/^\/api/', '', $_SERVER['REQUEST_URI']);

```

Comment:
There was accompanying text with this explaining it in the file. Should probably be readded